homework/index.js

```diff
@@ -30,13 +30,93 @@
     return elem;
   }
 
+  function renderRepository(leftContainer, repository) {
+    const tabel = createAndAppend('tabel', leftContainer, { id: 'tabel' });
+    const tr1 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr1, { class: 'col1_data', text: 'Repository:' });
+    createAndAppend('td', tr1, { class: 'col2_data', text: repository.name });
+
+    const tr2 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr2, { class: 'col1_data', text: 'Description:' });
+    createAndAppend('td', tr2, { class: 'col2_data', text: repository.description });
+
+    const tr3 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr3, { class: 'col1_data', text: 'Forks:' });
+    createAndAppend('td', tr3, { class: 'col2_data', text: repository.forks });
+
+    const tr4 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr4, { class: 'col1_data', text: 'Updated:' });
+    createAndAppend('td', tr4, { class: 'col2_data', text: repository.updated_at });
+  }
+
+  function renderContributors(rightContainer, url) {
+    fetchJSON(url, (err, contributors) => {
+      if (err !== null) {
+        createAndAppend('div', rightContainer, { text: err.message, class: 'alert-error' });
+      } else {
+        createAndAppend('p', rightContainer, { class: 'contributions', text: 'Contributions' });
+        const ul = createAndAppend('ul', rightContainer, { class: 'listOfcontributors' });
+        contributors.forEach(contributor => {
```

Comment:
The renderContributors looks good, the only thing I would improve here is extract the code inside the forEach into a separate function that could be called renderContributor, just to make renderContributors shorter and easier to read.